jvm-packages/xgboost4j-spark-gpu/pom.xml

```diff
@@ -23,9 +23,12 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-shade-plugin</artifactId>
                 <configuration>
+                  <createDependencyReducedPom>true</createDependencyReducedPom>
+                  <useDependencyReducedPomInJar>true</useDependencyReducedPomInJar>
                   <shadedArtifactAttached>false</shadedArtifactAttached>
                   <artifactSet>
                     <includes>
+                      <include>ml.dmlc:xgboost4j_${scala.binary.version}</include>
                       <include>ml.dmlc:xgboost4j-spark_${scala.binary.version}</include>
```

Comment:
Looks like xgboost4j-spark has assembled the xgboost4j?

Reply:
This is needed so that the reduced POM of (shaded) XGBoost4J-Spark-GPU doesn't include XGBoost4J.

Reply:
what does that mean? you mean just the pom didn't contain xgboost4j? or xgboost4j-spark-gpu.jar didn't contain xgboost4j in it?

Reply:
Oh, I see, thx